FlowCrypt/Core/Core Services/MailDestinations.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  MailDestinations.swift
+//  FlowCrypt
+//
+//  Created by Anton Kharchevskyi on 8/22/19.
+//  Copyright © 2019 FlowCrypt Limited. All rights reserved.
+//
+
+import Foundation
+
+enum MailDestination {
+    enum Gmail {
+        case trash
+
+        var path: String {
+            switch self {
+            case .trash: return "[Gmail]/Trash"
```

Comment:
For now OK, but in the future we will support more different email providers, and every email provider may call it something else :) ..